src/components/DetailsMenu/DetailsMenuCard/styles.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+import { InfoSprinkle } from "@leafygreen-ui/info-sprinkle";
+import { Subtitle, SubtitleProps } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+
+const DetailRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: ${size.m};
+`;
+
+const DetailName = styled(Subtitle)<SubtitleProps>`
+  font-size: ${size.s};
+  margin-right: ${size.xs};
+`;
+
+const LabelWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+interface DetailsLabelProps {
+  label: string;
+  children: string;
+}
+const DetailsLabel: React.FC<DetailsLabelProps> = ({ children, label }) => (
+  <LabelWrapper>
+    <DetailName>{children}</DetailName>
+    {/* @ts-expect-error https://jira.mongodb.org/browse/LG-3797 */}
+    <InfoSprinkle>{label}</InfoSprinkle>
```

Comment:
Slight preference for making the `triggerEvent` for all of these tooltips a click... sometimes when navigating to the preference you'd like to toggle a bunch of tooltips will pop up

Reply:
That doesn't appear to be an option on the InfoSprinkle component. It looks like there is a slight delay when hovering before the tooltip pops up which should avoid accidental toggles of it.
https://www.mongodb.design/component/info-sprinkle/example/

Reply:
Interesting, it actually does seem to work locally when I add `triggerEvent` but isn't documented. Up to you!